config/administrator.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+return array(
+
+	/**
+	 * Package URI
+	 *
+	 * @type string
+	 */
+	'uri' => 'admin',
+
+	/**
+	 *  Domain for routing.
+	 *
+	 *  @type string
+	 */
+	'domain' => '',
+
+	/**
+	 *  Middleware for admin routing.
+	 *
+	 *  @type array
+	 */
+	'middleware' => array(),
+
+	/**
+	 * Page title
+	 *
+	 * @type string
+	 */
+	'title' => 'Admin',
+
+	/**
+	 * The path to your model config directory
+	 *
+	 * @type string
+	 */
+	'model_config_path' => config_path('administrator'),
+
+	/**
+	 * The path to your settings config directory
+	 *
+	 * @type string
+	 */
+	'settings_config_path' => config_path('administrator/settings'),
+
+	/**
+	 * The menu structure of the site. For models, you should either supply the name of a model config file or an array of names of model config
+	 * files. The same applies to settings config files, except you must prepend 'settings.' to the settings config file name. You can also add
+	 * custom pages by prepending a view path with 'page.'. By providing an array of names, you can group certain models or settings pages
+	 * together. Each name needs to either have a config file in your model config path, settings config path with the same name, or a path to a
+	 * fully-qualified Laravel view. So 'users' would require a 'users.php' file in your model config path, 'settings.site' would require a
+	 * 'site.php' file in your settings config path, and 'page.foo.test' would require a 'test.php' or 'test.blade.php' file in a 'foo' directory
+	 * inside your view directory.
+	 *
+	 * @type array
+	 *
+	 * 	array(
+	 *		'E-Commerce' => array('collections', 'products', 'product_images', 'orders'),
+	 *		'homepage_sliders',
+	 *		'users',
+	 *		'roles',
+	 *		'colors',
+	 *		'Settings' => array('settings.site', 'settings.ecommerce', 'settings.social'),
+	 * 		'Analytics' => array('E-Commerce' => 'page.ecommerce.analytics'),
+	 *	)
+	 */
+	'menu' => array(
+        'members',
+        'Settings' => array('settings.site'),
+    ),
+
+	/**
+	 * The permission option is the highest-level authentication check that lets you define a closure that should return true if the current user
+	 * is allowed to view the admin section. Any "falsey" response will send the user back to the 'login_path' defined below.
+	 *
+	 * @type closure
+	 */
+	'permission'=> function()
+	{
+		return Auth::check() && Auth::user()->hasRole('editor');
```

Comment:
For reference
> Spencer: did you check if Auth::user() is null if not authenticated
http://laravel.io/forum/09-15-2015-laravel-5-can-not-get-authenticated-user-from-view

(This originally omitted the `Auth::check()`.)